### ISSUE TITLE
How to use phoenix-2014t_vs2t.yaml

### ISSUE THREAD

**Issue by consideringlee** [rzhao-zhsq/CV-SLT#4]
Hello, thank you very much for your work. However, I found that the results were skewed during the reproduction of the paper, and I wanted to reach the target level by tuning the parameters, but phoenix's BLEU-4 score has been maintained at around 26, so how can I solve this problem, and I would appreciate a reply,thank you very much!

**Comment by rzhao-zhsq** [comment]
@consideringlee Hi, you can check whether the pre-trained ckpts (s2g/g2t) are loaded correctly, then, we recommend reproducing the inherited baseline [MMTLB](https://arxiv.org/abs/2203.04287) first to tune hyper-params (e,g, learning rate, batch size) that match your hardware. In our case, the bleu-4 score we reproduced on MMTLB  is about but less than 28.00 on phoenix-14t.

Good luck!
